### DISCUSSION TITLE
Should we release 0.69.6?

### DISCUSSION THREAD

**Discussion by kelset** [reactwg/react-native-releases#31] [Patches]
Released, please refer to https://github.com/reactwg/react-native-releases/discussions/35

---

Current Release: [0.69.5](https://github.com/facebook/react-native/releases/tag/v0.69.5)

Conversations on this thread are limited:

- [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking).
- [qualified cherry-pick requests](https://reactnative.dev/contributing/release-faq#what-is-a-qualified-pick-request) of commits on main that [did not make the previous patch version](https://reactnative.dev/contributing/release-faq#how-do-i-know-if-my-fixfeature-is-in-a-certain-release).

Please include a link to the specific commit on main to be cherry-picked, for example: [facebook/react-native@bd2b7d6](https://github.com/facebook/react-native/commit/20b0eba581a00e5e7e300f6377379b836617c147)

In other words, if you cannot point to a particular commit on main, then your request likely belongs as a new issue.
If the issue is a [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking), please reference the issue here.

---

#### List of qualified picks

1. https://github.com/facebook/react-native/commit/7d42106d4ce20c644bda4d928fb0abc163580cee
2. https://github.com/facebook/react-native/commit/361b310bcc8dddbff42cf63495649291c894d661
3. https://github.com/facebook/react-native/commit/81564c1a3dae4222858de2a9a34089097f665e82
4. https://github.com/facebook/react-native/commit/3afef3c16702cefa5115b059a08741fba255b2db
5. https://github.com/facebook/react-native/commit/951538c080ef745da304fb308fa91d597e0dd98a

#### Local commits to backport to main

1.

**Comment by Kudo** [comment]
https://github.com/facebook/react-native/commit/3afef3c16702cefa5115b059a08741fba255b2db hopefully to have the fix when 0.69.6 would go for planning.

**Comment by AlirezaHadjar** [comment]
Hi.
Importing components from path alias isn't working on 0.69. but it works fine on 0.68. Are there any extra steps that I missed?
https://github.com/facebook/react-native/issues/34507
https://github.com/facebook/react-native/issues/34434

**Comment by tido64** [comment]
We should pick this Yoga crash fix: https://github.com/facebook/react-native/commit/361b310bcc8dddbff42cf63495649291c894d661

**Comment by retyui** [comment]
I would be great to cherry pick https://github.com/facebook/react-native/commit/951538c080ef745da304fb308fa91d597e0dd98a too

**Comment by sunnylqm** [comment]
this one plz https://github.com/facebook/react-native/commit/7d42106d4ce20c644bda4d928fb0abc163580cee

**Comment by Kudo** [comment]
https://github.com/facebook/react-native/commit/81564c1a3dae4222858de2a9a34089097f665e82 to fix hermes profiler

**Comment by pilotpirxie** [comment]
Would be great to have that fix for Xcode 14 included https://github.com/CocoaPods/CocoaPods/issues/11402#issuecomment-1201464693 
https://github.com/facebook/react-native/issues/34673

**Reply by tido64** [reply]
Hi @pilotpirxie, this thread is for cherry-picking fixes that have landed on `main` to fix current releases (see description in OP). That said, this looks like a bug in CocoaPods rather than react-native. The workarounds that have been mentioned can be temporarily applied to your `Podfile` to unblock yourself, although I would not recommend disabling code signing as it's required for deploying to your devices and publishing to the App Store.

**Reply by pilotpirxie** [reply]
Hi, ah I see. Thank you for the explanation.
